src/thunderbots/software/ai/world/team.cpp

```diff
@@ -76,15 +76,15 @@ void Team::removeExpiredRobots(const std::chrono::steady_clock::time_point times
 
 void Team::assignGoalie(unsigned int new_goalie_id)
 {
-    if (getRobotById(new_goalie_id))
+    /*if (getRobotById(new_goalie_id))
     {
         goalie_id = new_goalie_id;
     }
     else
     {
         throw std::invalid_argument(
             "Error: Assigning the goalie to a robot that is not a member of the team");
-    }
+    }*/
```

Comment:
I'm a bit confused, how is this change related?

Reply:
This exception is being unexpectedly thrown when running the AI, so this is a temporary fix. A proper fix will need to be investigated/implemented separately. I have a reminder to create a ticket for this.

Reply:
Oooh just realized this causes test failures. Going to prioritize a proper fix for this ASAP that should be merged first.

Reply:
Wut, scary.